processor/src/errors.rs

```diff
@@ -190,6 +191,9 @@ impl Display for ExecutionError {
                     "An operation expected a u32 value, but received {v} (error code: {err_code})"
                 )
             },
+            OutputStackOverflow(n) => {
+                write!(f, "Overflow table should be empty at the end of program execution, but has {n} rows")
```

Comment:
```suggestion
                write!(f, "The stack should have at most {STACK_TOP_SIZE} elements at the end of program execution, but had {} elements", STACK_TOP_SIZE + n)
```
Let's make this error message more stack-focused than "overflow table" focused